File: client/src/app/_models/cliente.ts
import { Usuario } from "./produto/usuario";

export interface Cliente {
    id:string;
    nome:string;
    email:string;
    telefone:string;
    endereco:string;
    tipoDocumento:string;
    numeroDocumento:string;
    dataNascimento:Date;
    idade:number;
    dataCadastro:Date;
    dataAtualizacao:Date;
    usuario:Usuario;
}

File: src/VirtualLibrary.Domain/Models/Cliente.cs
using System;
using System.Collections.Generic;

namespace VirtualLibrary.Domain.Models
{
    public class Cliente : Entity
    {
        public Guid? UsuarioId { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public string Telefone { get; set; }
        public string Endereco { get; set; }
        public string TipoDocumento { get; set; }
        public string NumeroDocumento { get; set; }
        public DateTime DataNascimento { get; set; }
        public DateTime DataCadastro { get; set; }
        public DateTime DataAtualizacao { get; set; }

        public Usuario Usuario { get; set; }
        /*Ef Relation*/
        public IEnumerable<Aluguer> Alugueres { get; set; }
    }
}